### PR TITLE
GUVNOR-2539: Guided Decision Table Editor: VersionRecordManager does not refresh when different documents are selected

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/GUVNOR-2539

### CODE REVIEW

File: uberfire-extensions/uberfire-commons-editor/uberfire-commons-editor-client/src/main/java/org/uberfire/ext/editor/commons/client/history/VersionRecordManager.java [hunk 1]
```diff
@@ -278,6 +287,7 @@ public void clear() {
         versions = null;
         pathToLatest = null;
         version = null;
+        versionMenuDropDownButton.resetVersions();
```
Comment (manstis) [inline uberfire-extensions/uberfire-commons-editor/uberfire-commons-editor-client/src/main/java/org/uberfire/ext/editor/commons/client/history/VersionRecordManager.java]: This is the *real* change.. everything else is formatting (the existing code was uuuugly so I tarted it around a bit)

**Comment by manstis** [comment]
@ederign @porcelli @csadilek Sorry, you're the only ones who would review UF PRs..

**Comment by porcelli** [comment]
thx for the highlight of real change ;)
+1

**Comment by csadilek** [comment]
+1

**Comment by manstis** [comment]
@porcelli @csadilek Would one of you mind merging? (I don't have access to UF)

**Pull request merged**